spec/models/note_spec.rb

```diff
@@ -0,0 +1,4 @@
+require "rails_helper"
+
+RSpec.describe Note, type: :model do
```

Comment:
Do we need this file if there are no specs for the model?

Reply:
Not really, but don't think it costs anything either. It also means it's there ready for someone if/when they need to add specs to the model.